lib/single_cov.rb

```diff
@@ -76,6 +76,26 @@ def assert_tested(files: glob('{app,lib}/**/*.rb'), tests: default_tests, untest
       end
     end
 
+    def assert_full_coverage(tests: default_tests, currently_complete: [], currently_complete_file: "")
+      complete = tests.select { |file| File.read(file) =~ /SingleCov.covered!(\s*|\s*\#.*)$/ }
+      missing_complete = currently_complete - complete
+      newly_complete = complete - currently_complete
+      errors = []
+
+      unless missing_complete.empty?
+        errors << "The following file(s) were marked as 100% SingleCov test coverage (had no `coverage:` option) and are no longer. " +
+        "Please extend test coverage in these files to maintain 100% coverage. #{missing_complete}"
+      end
+
+      unless newly_complete.empty?
+        errors << "The following files are newly at 100% SingleCov test coverage. " +
+        "Please add the following to #{currently_complete_file} to ensure 100% coverage is maintained moving forward. " +
```

Comment:
```suggestion
        "Please add them to #{caller(0..1)[1]} to ensure 100% coverage is maintained moving forward.\n" +
```

Reply:
The file path with the list of tests at 100% coverage is different from the file of method calling this test.  As such, I would like to preserve the code as written.  Thanks :)

Reply:
prefer a simpler api that ideally explains itself
- there is no doc for what currently_complete_file is
- users might want to use it without an extra file

so for example `message: nil` could be a good 3rd argument (like most asserts take) and then that can point the user to updating `currently_complete_file`

Reply:
can you address this too ? ^

Reply:
Great minds!  I was just working on implementing that feedback.  Changes are now pushed.

Thank you so much for your feedback - it leveled up this code and helped me learn :)